### ISSUE TITLE
Task is not showing in Sprint

### ISSUE THREAD

**Issue by SteinbrecherJ** [yanivsegev/VSTS-DropPlan#145]
We have created a task that is linked to a story, the story is displayed but the assigned task is not displayed in the DropPlan although it is assigned to the employee and is still in the valid sprint.

**Comment by yanivsegev** [comment]
Any task or bug assigned to the current sprint should be displayed on the plan - then, we enrich it with their parent info if it exists in the same sprint.
Do you see other tasks in the plan? Is there any difference between the shown and not shown tasks?

**Comment by SteinbrecherJ** [comment]
Yes, there are some Task in the Drop Plan yes but only this Task is not shown.



Mit freundlichen Grüßen - Best regards
Jörg Steinbrecher

[delaware logo]<https://www.delaware.pro/de-de/?utm_source=outlook&utm_medium=signature&utm_campaign=default-rebranding-signature-en&utm_content=homepage>
+49 6332 485 0
+49 151 161 32 396
***@***.******@***.***>
delaware Germany GmbH
Amerikastr. 15
66482 Zweibruecken, Germany
Commercial Register:
Amtsgericht Zweibrücken, HRB 30963
Managing Director: Michael Lauterbach


Blog<https://www.delaware.pro/de-de/blogs> • LinkedIn<https://www.linkedin.com/company/delaware-germany/> • Facebook<https://www.facebook.com/DelawareGermany>

legal disclaimer<https://www.delaware.pro/en-de/legal/privacy> - think before you print

This e-mail and any attachment are confidential and intended solely for the use of the individual to whom it is addressed. If you are not the intended recipient, please telephone or email the sender and delete this message and any attachment from your system. Unauthorized publication, use, dissemination, forwarding, printing or copying of this e-mail and its associated attachments is strictly prohibited.

Von: yanivsegev ***@***.***>
Gesendet: Freitag, 2. August 2024 13:31
An: yanivsegev/VSTS-DropPlan ***@***.***>
Cc: Steinbrecher, Jörg ***@***.***>; Author ***@***.***>
Betreff: Re: [yanivsegev/VSTS-DropPlan] Task is not showing in Sprint (Issue #145)

ATTENTION ─ This is an external email. Do not click links or open attachments unless you recognize the sender and know the content is safe.


Any task or bug assigned to the current sprint should be displayed on the plan - then, we enrich it with their parent info if it exists in the same sprint.
Do you see other tasks in the plan? Is there any difference between the shown and not shown tasks?

**Comment by yanivsegev** [comment]
Is there any difference between the shown and not shown tasks? 
Do both of them have the same icon / type / sprint?

**Comment by sgtwilko** [comment]
@SteinbrecherJ can you double check that the Task has the exact same sprint assigned as the parent.
It's not unusual for the Parent story to be in one sprint, and the the child task to be in a different one.
